packages/utils/models.js

```diff
@@ -4,13 +4,31 @@ const ThisStore = {
   _storedModel: undefined,
 };
 
+function VerifyMethodOnSystemWrapper(method) {
+  try {
+    /* Necessary because the circular dependency with simulator */
+    const _system = window.$System || window.System;
+
+    return _system[method]();
+  } catch (error) {
+    if (__DEV__) console.error(error);
+  }
+}
```

Comment:
É bom os retornos serem explícitos aqui. 
```suggestion
/**
 * @return {boolean}
 */
function VerifyMethodOnSystemWrapper(method) {
  try {
    /* Necessary because the circular dependency with simulator */
    const _system = window.$System || window.System;

    return _system[method]();
  } catch (error) {
    if (__DEV__) console.error(error);
    return false;
  }
}
```

Reply:
ok